### PR TITLE
[UI] Close to Collateral token in AAVE + Payback debt, withdraw collateral

### PR DESCRIPTION
**Pull request by marcinciarka**
# [[UI] Close to Collateral token in AAVE](https://app.shortcut.com/oazo-apps/story/7003/ui-close-to-collateral-token-in-aave)
# [[UI] Payback debt, withdraw collateral (no multiplying)](https://app.shortcut.com/oazo-apps/story/6815/ui-payback-debt-withdraw-collateral-no-multiplying)
# [[UI] Deposit Collateral, borrow debt (no multiplying)](https://app.shortcut.com/oazo-apps/story/6814/ui-deposit-collateral-borrow-debt-no-multiplying)
  
## Changes 👷‍♀️
- tbd  
## How to test 🧪
- tbd

### CODE REVIEW

File: features/aave/manage/sidebars/SidebarManageAaveVault.tsx [hunk 1]
```diff
@@ -223,6 +326,63 @@ export function SidebarManageAaveVault() {
     return isLoading(state)
   }
 
+  const dropdownConfig: SidebarSectionHeaderDropdown = {
```
Comment (jakubswierczek) [inline features/aave/manage/sidebars/SidebarManageAaveVault.tsx:329]: maybe move it to another function?

File: features/aave/manage/sidebars/SidebarManageAaveVault.tsx [hunk 2]
```diff
@@ -100,37 +122,117 @@ function GetReviewingSidebarProps({
...
+                action: () => curry(updateDebtTokenAction)(action),
+              }))}
+            />
+            <Text as="p" variant="paragraph3" sx={{ color: 'neutral80' }}>
+              Manage debt
+            </Text>
+            <VaultActionInput
+              action="Enter"
+              currencyCode={debt}
+              tokenUsdPrice={new BigNumber(150)}
```
Comment (jakubswierczek) [inline features/aave/manage/sidebars/SidebarManageAaveVault.tsx:207]: change it to real price

File: features/aave/manage/sidebars/SidebarManageAaveVault.tsx [hunk 3]
```diff
@@ -100,37 +122,117 @@ function GetReviewingSidebarProps({
...
+              maxAmountLabel={'Balance'}
```
Comment (jakubswierczek) [inline features/aave/manage/sidebars/SidebarManageAaveVault.tsx:208]: Max Amount will be different depending on multiply and locked value in position

File: features/aave/common/BaseAaveContext.ts [hunk 4]
```diff
@@ -16,10 +16,17 @@ import { TransactionStateMachineResultEvents } from '../../stateMachines/transac
 import { TransactionParametersStateMachineResponseEvent } from '../../stateMachines/transactionParameters'
 import { UserSettingsState } from '../../userSettings/userSettings'
 import { AaveProtocolData } from '../manage/services'
+import {
+  ManageCollateralActionsEnum,
+  ManageDebtActionsEnum,
+} from '../manage/sidebars/SidebarManageAaveVault'
```
[the remark this answers is not in the document]
Comment (marcinciarka) [reply features/aave/common/BaseAaveContext.ts:22]: Moved to strategy config.

File: features/aave/manage/sidebars/SidebarManageAaveVault.tsx (hunk 2 above)
Comment (marcinciarka) [reply features/aave/manage/sidebars/SidebarManageAaveVault.tsx:199]: Removed that (not needed after all)

**Comment by marcinciarka** [comment]
Moved to https://github.com/OasisDEX/oasis-borrow/pull/1808

**Pull request closed without merging**